### ISSUE TITLE
Support for Logitech G815

### ISSUE THREAD

**Issue by K1ngjulien** [libratbag/piper#478]
**Is your feature request related to a problem? Please describe.**
I've got a Logitech G815. libratbag detects it since the last update and it also shows up in piper.
Unfortunately, the software thinks the first two macro keys are left and right mouse button, so I can't change them at all and the lighting control doesn't work either.
Furthermore, there's no preview image being loaded.

**Describe the solution you'd like**
I'd like to help add support for this keyboard.
I can start by drawing a image for it in inkscape and then continue with the code but I'd probably need some help with that :)

**Additional context**
![Screenshot_20200226_175523](https://user-images.githubusercontent.com/16562333/75367824-2c192800-58c1-11ea-975e-b20b376e903d.png)

**Comment by K1ngjulien** [comment]
Do you guys even want to support keyboads?

**Comment by K1ngjulien** [comment]
Ok I just figured out that the lighting does work, but I have to press one of the "M-Keys" to refresh the lighting after hitting apply

**Comment by FFY00** [comment]
Yes, unfortunately you need to press one of the M-Keys after saving the changes to the device. This behavior comes from the keyboard, I haven't found any way to get around it.

Your PR looks great! I will test it when I get home.

**Comment by agolks** [comment]
will it be possible to set colors per key?
currently i need to use g810-led instead of piper to get custom colors.

**Comment by agolks** [comment]
will it be possible to set colors per key?
currently i need to use g810-led instead of piper to get custom colors.

**Comment by agolks** [comment]
will it be possible to set colors per key?
currently i need to use g810-led instead of piper to get custom colors.

**Comment by agolks** [comment]
will it be possible to set colors per key?
currently i need to use g810-led instead of piper to get custom colors.

**Comment by agolks** [comment]
will it be possible to set colors per key?
currently i need to use g810-led instead of piper to get custom colors.

**Comment by agolks** [comment]
will it be possible to set colors per key?
currently i need to use g810-led instead of piper to get custom colors.

**Comment by agolks** [comment]
will it be possible to set colors per key?
currently i need to use g810-led instead of piper to get custom colors.

**Comment by agolks** [comment]
will it be possible to set colors per key?
currently i need to use g810-led instead of piper to get custom colors.

**Comment by agolks** [comment]
will it be possible to set colors per key?
currently i need to use g810-led instead of piper to get custom colors.

**Comment by matrixik** [comment]
I have piper 0.5.1 keyboard and G815 image is loaded properly but buttons G1 and G2 are still detected as left and right mouse button without ability to change them.
